msnoise/api.py

```diff
@@ -280,7 +290,7 @@ def get_filters(session, all=False):
     :param all: Returns all filters from the database if True, or only filters
         where `used` = 1 if False (default)
 
-    :rtype: list of :class:`~msnoise.msnoise_table_def.Filter`
+    :rtype: list of :class:`~msnoise.msnoise_table_def.declare_tables.Filter`
```

Comment:
general question: did you try generating the HTML documentation to see if all pass?

Reply:
No, sorry. I also have a doubt this _msnoise.msnoise_table_def.declare_tables.Filter_ reference, but I have no experience with rst nor sphinx, and did not know how to deal with this. So I shamefully let this correction for later, because I wouldn't have to time to dig this this week :/

Reply:
hehe ok... will checkout your branch and check that